plugin/pkg/managedseed/validator/admission.go

```diff
@@ -218,6 +218,9 @@ func (v *ManagedSeed) Admit(ctx context.Context, a admission.Attributes, o admis
 	if !gardencorehelper.ShootWantsVerticalPodAutoscaler(shoot) {
 		return apierrors.NewInvalid(gk, managedSeed.Name, append(allErrs, field.Invalid(shootNamePath, managedSeed.Spec.Shoot.Name, "shoot VPA has to be enabled for managed seeds")))
 	}
+	if shoot.IsWorkerless() {
+		return apierrors.NewInvalid(gk, managedSeed.Name, append(allErrs, field.Invalid(shootNamePath, managedSeed.Spec.Shoot.Name, "shoot workers can not be empty for managed seeds")))
```

Comment:
Shoot has to be already present to create managed seed.
```suggestion
		return apierrors.NewInvalid(gk, managedSeed.Name, append(allErrs, field.Invalid(shootNamePath, managedSeed.Spec.Shoot.Name, "workerless shoot can't be used to create managed seed")))
```